pinot-controller/src/main/java/com/linkedin/pinot/controller/ControllerConf.java

```diff
@@ -265,6 +267,18 @@ public void setStatusCheckerFrequencyInSeconds(int statusCheckerFrequencyInSecon
     setProperty(STATUS_CHECKER_FREQUENCY_IN_SECONDS, Integer.toString(statusCheckerFrequencyInSeconds));
   }
 
+  public int getRelocationManagerFrequencyInMinutes() {
+    if (containsKey(REALTIME_SEGMENTS_RELOCATION_MANAGER_FREQUENCY_IN_MINUTES)) {
+      return Integer.parseInt((String) getProperty(REALTIME_SEGMENTS_RELOCATION_MANAGER_FREQUENCY_IN_MINUTES));
```

Comment:
Can we have the config similar to that in KafkaHighLevelStreamProviderConfig (using PeriodFormatter)? thanks

Reply:
Changed it. I was trying to keep it consistent with all the other frequency values in the config